Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:16.04
```

Comment:
I think this is likely to be a better approach than the `FROM python:2.7.12` base image used in https://github.com/StreisandEffect/streisand/pull/442 :+1:

Reply:
TBH, I'm just churning these out looking for the easiest path from A to B. Use of Docker for this is pretty heavy-duty, but in return, we get an entire OS we can scribble on; this is similar to the assumptions made in the `cloud-init` stuff.

Python virtualenv is sorta the next step down.

Yes, this stuff needs to migrate into a subdirectory.

Reply:
I don't agree on basing on Ubuntu. Python base is the shortest path where no need to rework a lot of stuff. virtualenv is nothing to look at when using docker, the whole container is a virtual env by itself.